mypy/nodes.py

```diff
@@ -781,6 +781,8 @@ class FuncDef(FuncItem, SymbolNode, Statement):
         "abstract_status",
         "original_def",
         "deco_line",
+        "is_trivial_body",
+        "is_mypy_only",
```

Comment:
Should this be added to `FUNCDEF_FLAGS`?

Reply:
Yes, good catch, we should serialize this